R/coord-sf.R

```diff
@@ -127,8 +127,16 @@ CoordSf <- ggproto("CoordSf", CoordCartesian,
 
   setup_panel_params = function(self, scale_x, scale_y, params = list()) {
     # Bounding box of the data
-    x_range <- scale_range(scale_x, self$limits$x, self$expand)
-    y_range <- scale_range(scale_y, self$limits$y, self$expand)
+    x_range <- expand_limits_scale(
+      scale_x,
+      expand_default(scale_x, expand = self$expand),
```

Comment:
Maybe save `expand_default(scale_x, expand = self$expand)` in variable so the code resembles the other coords a bit more closely?